cmd/controller/main.go

```diff
@@ -416,3 +426,22 @@ func parseClusterDNSIPs(s string) ([]net.IP, error) {
 	}
 	return ips, nil
 }
+
+// return label selector to only process machines with a matching machine.k8s.io/controller label
+func labelSelector(workerName string) func(*metav1.ListOptions) {
+	return func(options *metav1.ListOptions) {
+		var req *labels.Requirement
+		var err error
+		if workerName == "" {
+			if req, err = labels.NewRequirement(controllerNameLabelKey, selection.DoesNotExist, nil); err != nil {
```

Comment:
Is it possible to allow both `selection.DoesNoetExist` and empty string here?

Reply:
It's not possible to select for both at the same time.